tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md

```diff
@@ -418,6 +418,7 @@ AccuracyChecker supports following set of adapters:
     * `keep_shape` - allow keeping shape of predicted multi dimension array (Optional, default False).
 * `multi_output_regression` - converting raw output features to `RegressionPrediction` for regression with gt data.
   * `output` - list of target output names.
+  * `online` - if online is True, adapter will ignore the batch-size. Default is False.
```

Comment:
for me, it is not clear why online means ignore batch on the tool side, could you please give more descriptive name for parameter,
`ignore_batch` will be ok for me

Reply:
oh, online is such ambiguous. I'll correct it.